services/directoryServices/MediaDirectoryService.js

```diff
@@ -21,9 +39,9 @@ class MediaDirectoryService {
     const { private: isPrivate } = await this.gitHubService.getRepoInfo(
       reqDetails
     )
-    const files = await this.baseDirectoryService.list(reqDetails, {
-      directoryName,
-    })
+    const files = await this.listWithDefault(reqDetails, { directoryName })
+    console.log(files)
```

Comment:
Unused `console.log`

Reply:
Removed in https://github.com/isomerpages/isomercms-backend/pull/337/commits/8cb9f79d097d7814c053127c731ef1f7206747b9